config.yml.example

```diff
@@ -26,7 +26,7 @@ auth_file: "/home/git/.ssh/authorized_keys"
 
 # Redis settings used for pushing commit notices to gitlab
 redis:
-  bin: /usr/bin/redis-cli
+  bin: /home/git/gitlab-shell/bin/dummy-redis-cli
```

Comment:
If this script is an empty file, wouldn't it be possible to simply disable it altogether?!